quint/src/repl.ts

```diff
@@ -516,6 +558,14 @@ function tryEvalHistory(out: writer, state: ReplState): boolean {
   return true
 }
 
+// Try to evaluate the expression in a string and print it, if successful.
+// After that, clear the recorded stack.
+function tryEvalAndClear(out: writer, state: ReplState, newInput: string): boolean {
```

Comment:
From the name, it is unclear that only the recorder is cleared after evaluation. Can we try to come up with a better name here? I thought of:

```suggestion
function tryEvalWithoutRecording(out: writer, state: ReplState, newInput: string): boolean {
```